### PR TITLE
Condensed PHP SDK to one page

### PR DESCRIPTION
**Pull request by MimiDumpling**
Added more information onto the PHP index page to make it easier for a developer to get what they need in one place for the PHP SDK.

### CODE REVIEW

File: src/collections/_documentation/platforms/php/index.md [hunk 1]
```diff
@@ -2,43 +2,262 @@
...
 
-This documentation goes over some PHP specific things such as integrations to
-frameworks.
+Sentry captures data by using an SDK within your application’s runtime. These are platform-specific and allow Sentry to have a deep understanding of how your application works.
 
-## Integrations
+To install the PHP SDK, you need to be using Composer in your project. For more details about Composer, see the [Composer documentation](https://getcomposer.org/doc/).
+
+```bash
+composer require sentry/sdk:2.0.3
```
Comment (HazAT) [inline src/collections/_documentation/platforms/php/index.md]:
```suggestion
composer require sentry/sdk:{% sdk_version sentry.php.sdk %}
```

File: src/collections/_documentation/platforms/php/index.md [hunk 2]
```diff
@@ -2,43 +2,262 @@
...
+## Capturing Errors
+
+In PHP you can either capture a caught exception or capture the last error with captureLastError.
+
+```php
+try {
+    $this->functionFailsForSure();
+} catch (\Throwable $exception) {
+    Sentry\captureException($exception);
+}
```
Comment (HazAT) [inline src/collections/_documentation/platforms/php/index.md]:
```suggestion
}

// OR

Sentry\captureLastError();
```

File: src/collections/_documentation/platforms/php/index.md [hunk 3]
```diff
@@ -2,43 +2,262 @@
...
+## Releases
+
+A release is a version of your code that you deploy to an environment. When you give Sentry information about your releases, you unlock many new features:
+
+- Determine the issue and regressions introduced in a new release
+- Predict which commit caused an issue and who is likely responsible (suspect commits)
+- Resolve issues by including the issue number in your commit message
+- Receive email notifications when your code gets deployed
+
+After configuring your SDK, setting up releases is a 2-step process:
```
Comment (HazAT) [inline src/collections/_documentation/platforms/php/index.md]: Add this example here:
```
```php
Sentry\init([
  'dsn' => '___PUBLIC_DSN___',
  'release' => 'your release name'
]);
```
```

File: src/collections/_documentation/platforms/php/index.md [hunk 4]
```diff
@@ -2,43 +2,262 @@
...
+
+[**Fingerprint**](#setting-the-fingerprint)
+
+: A value used for grouping events into issues
+
+[**Unstructured Extra Data**](#extra-context)
+
+: Arbitrary unstructured data which the Sentry SDK stores with an event sample
+
+### Setting Context
```
Comment (HazAT) [inline src/collections/_documentation/platforms/php/index.md]: This section has to go for now until we ship it.

File: src/collections/_documentation/platforms/php/index.md [hunk 5]
```diff
@@ -2,43 +2,262 @@
...
+
+### Extra Context
+
+```php
+Sentry\configureScope(function (Sentry\State\Scope $scope): void {
+  $scope->setExtra('character_name', 'Mighty Fighter');
+});
+```
+
+### Unsetting Context
```
Comment (HazAT) [inline src/collections/_documentation/platforms/php/index.md]: This section has to go for now until we ship it.

File: src/collections/_documentation/platforms/php/index.md [hunk 6]
```diff
@@ -2,43 +2,262 @@
...
+]);
+```
+
+For more information, see:
+
+- [Full documentation on Filtering Events]({%- link _documentation/error-reporting/configuration/filtering.md -%})
+- [Manage Your Flow of Errors Using Inbound Filters](https://blog.sentry.io/2017/11/27/setting-up-inbound-filters)
+
+
+### Integrations
```
Comment (HazAT) [inline src/collections/_documentation/platforms/php/index.md]: This seems disconnected here, we either need to list the integrations we have below or move it somewhere else
We have the default integrations on the current docs that can go beneath it or link to them.

**Comment by MimiDumpling** [comment]
@HazAT made the suggested changes. ;) Ready to make more changes, if needed.

**Pull request merged**